crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -23,7 +23,8 @@ class ApplicationController < ActionController::Base
   rescue_from Crowbar::Error::NotFound, with: :render_not_found
   rescue_from Crowbar::Error::ChefOffline, with: :chef_is_offline
 
-  before_filter :enable_profiler, if: Proc.new { ENV["ENABLE_PROFILER"] == "true" }
+  before_filter :enable_profiler, if: proc { ENV["ENABLE_PROFILER"] == "true" }
+  before_filter :enforce_installer, unless: proc { Crowbar::Installer.successful? || ENV["RAILS_ENV"] == "test" }
```

Comment:
Metrics/LineLength: Line is too long. [113/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)